### PR TITLE
update-submodule: return sha and sha7 in action's outputs

### LINKED ISSUE #40: update-submodule: return SHA of created commit
It can be useful to get the SHA of the commit that this action created in the target repo.

Proposal: add two output variables:

```yml
sha: c5e10dfd6ec566d640e974986b8fbacaf717ae15
sha7: c5e10df
```

### PR DESCRIPTION
**Pull request by NickVolynkin**
See https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#outputs-for-composite-actions.

To access the output values, assign an id to the step that is using this action.
Then get the values like this: `${{ steps.<step-id>.outputs.sha }}`.

Resolves #40

---

Testing:  the following action step

```yml
      - name: Dump actions/update-submodule outputs
        run: echo '${{ toJSON(steps.create-branch.outputs) }}'
```

shows

```
{
  "sha": "846617ad9dad04f335feb24792ec57ce37200b62",
  "sha7": "846617a"
}
```

### CODE REVIEW

**Review by slavakirichenko — APPROVED** [review]

File: update-submodule/README.md [hunk 1]
```diff
@@ -89,6 +99,7 @@ jobs:
   submodule-update:
     name: Submodule update
     runs-on: docker
+    id: submodule-update
```
Comment (ylobankov) [inline update-submodule/README.md:102]: Looks wrong. ID should be inside the step below.

File: update-submodule/action.yml [hunk 2]
```diff
@@ -65,6 +65,13 @@ inputs:
     description: 'The message for the commit'
     default: 'Update submodule'
     required: false
+outputs:
+  sha:
+    description: "Full SHA1 of resulting commit"
```
Comment (ylobankov) [inline update-submodule/action.yml:70]: Please use single quotes for consistency with other parameters.

File: update-submodule/action.yml [hunk 3]
```diff
@@ -82,6 +89,7 @@ runs:
         fetch-depth: 0
 
     - name: Create the new branch and push changes
+      id: create-branch
```
Comment (ylobankov) [inline update-submodule/action.yml:92]: Shouldn't be it `submodule-update`? `create-branch` sounds weird.

Comment (ylobankov) [reply update-submodule/action.yml:92]: Although, I am okay. Please ignore.

File: update-submodule/README.md (hunk 1 above)
Comment (NickVolynkin) [reply update-submodule/README.md]: fixed

File: update-submodule/action.yml (hunk 2 above)
Comment (NickVolynkin) [reply update-submodule/action.yml]: fixed

File: update-submodule/action.yml (hunk 3 above)
Comment (NickVolynkin) [reply update-submodule/action.yml:92]: there are two steps: create branch, create pr.

**Review by ylobankov — APPROVED** [review]
LGTM

**Pull request merged**